lib/rangeclient.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rubygems'
 require 'net/http'
-require 'uri'
+require 'erb'
```

Comment:
Don't we want to keep `require 'uri'` instead of `erb` because we are no not using `ERB::Util.url_encode` ?

Reply:
Yep, 'tis fixed now.